### DISCUSSION TITLE
vkey or hkey without extra lines or rows in the blank border?

### DISCUSSION THREAD

**Discussion by laixintao** [Textualize/textual#3251] [Q&A] [answered]
I got one more question:

Is there a way to make a border of `vkey` without extra lines?

Currently:

```
|   |
| A |
|   |
```

I think it is because we need to display the title in border in some reused code with other borders, so that we must leave the extra 2 more lines?

But in my case, I want to show as much information as possible, with 4 items. So I do want a `vkey` border between each of them, but I do not want to waste 2 lines.

Can we have more css attrs like `border-bottom` `border-top` so that I can explicitly set them to `none`?

**Comment by willmcgugan** [comment, marked as answer]
> Can we have more css attrs like border-bottom border-top so that I can explicitly set them to none?

Those already exist in Textual. But those extra lines would only appear if your content is 3 lines high. Maybe set `padding: 0 1;`
